src/tests/presets/test_account.cairo

```diff
@@ -55,60 +55,61 @@ fn setup_dispatcher_with_data(data: Option<@SignedTransactionData>) -> AccountAB
 //
 
 #[test]
-#[available_gas(2000000)]
 fn test_constructor() {
     let mut state = Account::contract_state_for_testing();
     Account::constructor(ref state, PUBKEY);
 
     assert_only_event_owner_added(PUBKEY, ZERO());
 
-    assert(Account::PublicKeyImpl::get_public_key(@state) == PUBKEY, 'Should return PUBKEY');
-    assert(Account::SRC5Impl::supports_interface(@state, ISRC5_ID), 'Should implement ISRC5');
-    assert(Account::SRC5Impl::supports_interface(@state, ISRC6_ID), 'Should implement ISRC6');
+    let public_key = Account::PublicKeyImpl::get_public_key(@state);
+    assert_eq!(public_key, PUBKEY);
+
+    let supports_isrc5 = Account::SRC5Impl::supports_interface(@state, ISRC5_ID);
+    let supports_isrc6 = Account::SRC5Impl::supports_interface(@state, ISRC6_ID);
+    assert!(supports_isrc5);
+    assert!(supports_isrc6);
```

Comment:
why not asserting right after each query? this is the least important optimization but we can save us a call in case isrc5 fails early -- that's actually a guideline we try to follow in our contracts: fail early and loud

Reply:
Good point, I will update it.